src/Discussion/Search/Driver/MySqlDiscussionDriver.php

```diff
@@ -9,25 +9,34 @@
  * file that was distributed with this source code.
  */
 
-namespace Flarum\Discussion\Search\Fulltext;
+namespace Flarum\Discussion\Search\Driver;
 
 use Flarum\Post\Post;
+use Flarum\Discussion\Discussion;
 
-class MySqlFulltextDriver implements DriverInterface
+class MySqlDiscussionDriver implements DriverInterface
 {
     /**
      * {@inheritdoc}
      */
     public function match($string)
     {
-        $discussionIds = Post::where('type', 'comment')
+        $contentDiscussionIds = Post::where('type', 'comment')
             ->whereRaw('MATCH (`content`) AGAINST (? IN BOOLEAN MODE)', [$string])
             ->orderByRaw('MATCH (`content`) AGAINST (?) DESC', [$string])
             ->lists('discussion_id', 'id');
+        $titleDiscussionIds = Discussion::where('is_approved', 1)
+            ->whereRaw('MATCH (`title`) AGAINST (? IN BOOLEAN MODE)', [$string])
+            ->orderByRaw('MATCH (`title`) AGAINST (?) DESC', [$string])
+            ->lists('id');
```

Comment:
This isn't possible because the `discussions` table doesn't have a `content` column, as it references a post with `start_post_id`.

This causes the following error:

```php
PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'content' in 'where clause'
```